### PR TITLE
Spring cinema hw01 - Configure Spring in Cinema project

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

**Review by matviichuk92 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -24,6 +25,33 @@
...
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.11.3</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
```
Comment (RomanDubovskyi) [inline pom.xml]: seems that we have no jsp here

File: src/main/java/vic/cinema/Main.java [hunk 2]
```diff
@@ -15,10 +14,10 @@
 import vic.cinema.service.MovieService;
 import vic.cinema.service.MovieSessionService;
 import vic.cinema.service.OrderService;
-import vic.cinema.service.ShoppingCartService;
+import vic.cinema.service.ShoppingCartService;*/
 
 public class Main {
-    private static Injector injector = Injector.getInstance("vic.cinema");
+   /* private static Injector injector = Injector.getInstance("vic.cinema");
```
Comment (RomanDubovskyi) [inline src/main/java/vic/cinema/Main.java]: do we need commented code here?

File: pom.xml (hunk 1 above)
Comment (Victoria-Moiseienko) [reply pom.xml]: Done

File: src/main/java/vic/cinema/Main.java (hunk 2 above)
Comment (Victoria-Moiseienko) [reply src/main/java/vic/cinema/Main.java]: Done

File: pom.xml [hunk 3]
```diff
@@ -36,12 +36,6 @@
...
             <version>2.11.3</version>
         </dependency>
-        <dependency>
-            <groupId>javax.servlet.jsp</groupId>
-            <artifactId>javax.servlet.jsp-api</artifactId>
-            <version>2.3.1</version>
-            <scope>provided</scope>
-        </dependency>
         <dependency>
             <groupId>javax.servlet.jsp.jstl</groupId>
```
Comment (RomanDubovskyi) [inline pom.xml]: and what about this dependency?

**Review by RomanDubovskyi — APPROVED** [review]
remove one dependency

File: pom.xml (hunk 3 above)
Comment (Victoria-Moiseienko) [reply pom.xml]: Removed. Done

**Pull request merged**